ztp/kube-compare-reference/default_value.yaml

```diff
@@ -75,6 +75,53 @@ optional_ptp_config_PtpConfigDualCardGmWpc:
   captureGroup_defaults:
     priority2: 128
     domainNumber: 24
+optional_ptp_config_PtpConfigThreeCardGmWpc:
+- spec:
+    profile:
+    - plugins:
+        e810:
+          settings:
+            LocalMaxHoldoverOffSet: 1500
+            LocalHoldoverTimeout: 14400
+            MaxInSpecOffset: 1500
+          pins:
+            $iface_timeTx1:
+              SMA1: 2 1
+              SMA2: 2 2
+              U.FL1: 0 1
+              U.FL2: 0 2
+            $iface_timeTx2:
+              SMA1: 1 1
+              SMA2: 0 2
+              U.FL1: 0 1
+              U.FL2: 0 2
+            $iface_timeTx3:
+              SMA1: 1 1
+              SMA2: 0 2
+              U.FL1: 0 1
+              U.FL2: 0 2
+      ptpClockThreshold:
+        holdOverTimeout: 5
+        maxOffsetThreshold: 100
+        minOffsetThreshold: -100
```

Comment:
This threshold is not aligned with the MaxInSpecOffset at line 86, which is 1500ns. It will create a problem when recovering from holdover.
Let's assume you are in holdover and reached the offset of 1000ns, still below MaxInSpecOffset, and then the GNSS reception recovers. When it happens, we move to Locked where ptpClockThreshold guards the offset. The time error we have accumulated in holdover (1000ns) might not be immediately fixed, and we will cross ptpClockThreshold and move to Free-run
Described here:
https://issues.redhat.com/browse/OCPBUGS-51311